### PR TITLE
Neural Networks

### PR DESCRIPTION
**Pull request by DanishAlsayed**

### CODE REVIEW

File: results.csv [hunk 1]
```diff
@@ -1,4 +1,6 @@
...
-AdaBoost Classifier,51.6,Date,"Preliminary, 80/20 randomly sliced"
-Random Forest Classifier,51.2,Date,"Preliminary, 80/20 randomly sliced"
-SVM, 51.2,Date,"Preliminary, 80/20 first 80% for training and last 20% for testing"
\ No newline at end of file
+Model,%age correct,feature excluded,comments
+AdaBoost Classifier,51.6,Date,"Preliminary, 80/20 randomly sliced"
+Random Forest Classifier,51.2,Date,"Preliminary, 80/20 randomly sliced"
+SVM,51.2,Date,"Preliminary, 80/20 first 80% for training and last 20% for testing"
+Feedforward Neural Network,53.2,Date,"Preliminary, 80/20 randomly sliced and balancing. But the accuracy fluctuates due to the random initialisation in neural network"
+LSTM (Regression),Validation Loss: 0.23,Date,"Preliminary, First 1000 days are for change, the remaining days are for validation. Randomly sliced."
```
Comment (DanishAlsayed) [inline results.csv]: Pleaseadd the %age accuracy in that column and add the validation loss in the comments section or make a new column for it if you wish

**Comment by DanishAlsayed** [comment]
Why did you add the technical indicators like momentum etc? Did you use those? Can you also fit your model based on the features in `data_normalized_1d_5y.csv` first. Then we will add more features/techincal indicators and refit all the models to see how things improve.

File: results.csv (hunk 1 above)
Comment (oscarko318) [reply results.csv]: Added

**Comment by oscarko318** [comment]
> Why did you add the technical indicators like momentum etc? Did you use those? Can you also fit your model based on the features in `data_normalized_1d_5y.csv` first. Then we will add more features/techincal indicators and refit all the models to see how things improve.
